docker/build.sh

```diff
@@ -119,6 +126,7 @@ build_images() {
         --set "*.args.BASE_IMAGE=$base_image" \
         --set "*.args.SETUP_ARGS=$setup_args" \
         --set "*.args.LIB_DIR=$lib_dir" \
+        --set "base.tags=ghcr.io/autowarefoundation/autoware:latest-base" \
```

Comment:
I'm not sure but do you need `$image_name_suffix`?
```suggestion
        --set "base.tags=ghcr.io/autowarefoundation/autoware:latest-base$image_name_suffix" \
```

Reply:
As the `base `image does not have cuda extensions, we don't need to add `image_name_suffix`